scss/_navbar.scss

```diff
@@ -457,6 +447,25 @@
         .navbar-collapse {
           display: flex !important; // stylelint-disable-line declaration-no-important
```

Comment:
Can work without the important statement

Reply:
It comes from Bootstrap so let's keep it as-is if you're fine with it!